pkg/gui/whitespace-toggle.go

```diff
@@ -0,0 +1,21 @@
+package gui
+
+func (gui *Gui) toggleWhitespaceInDiffView() error {
```

Comment:
given that we want to remove the call of `setIgnoreWhitespaceFlag` in staging_panel.rb, we'll be left with `setIgnoreWhitespaceFlag` only being called from `toggleWhitespaceInDiffView`. As such, I would merge those two functions.